### DISCUSSION TITLE
Upload组件二次封装问题

### DISCUSSION THREAD

**Discussion by Lind-pro** [alibaba/formily#2329] [Q&A 2.x]
Upload组件的value是一个对象。如果只想获取上传文件后的URL应该怎么进行二次封装改造呢？

**Comment by simplast** [comment]
同问，我们上传是单独的接口，会返回文件key，表单提交的时候只需要提交文件key，这个取值要怎么封装

**Reply by janryWang** [reply]
文档里本身就是二次封装的例子，可以直接拦截value/onChange对数据做一次转换
